src/main/java/org/rocstreaming/roctoolkit/RocReceiver.java

```diff
@@ -228,7 +243,11 @@ public void setMulticastGroup(Slot slot, Interface iface, String ip) throws Ille
         Check.notNull(slot, "slot");
         Check.notNull(iface, "iface");
         Check.notEmpty(ip, "ip");
+
+        LOGGER.log(Level.FINE, "roc_receiver set multicast group started, ptr={0}, slot={1}, iface={2}, ip={3}",
+                new Object[]{getHexPtr(), slot.getValue(), iface, ip});
```

Comment:
nit: maybe add Slot.toString and remove getValue here?

Reply:
done